src/components/autosuggest/code.list.searcher.js

```diff
@@ -6,7 +6,16 @@ export default function queryJson(query, data, searchFields) {
     threshold: 0.2,
     location: 1,
     distance: 1000,
-    keys: [searchFields, 'tags'],
+    keys: [
+      {
+        name: searchFields,
+        weight: 0.2,
```

Comment:
![image](https://user-images.githubusercontent.com/42928680/132224691-9454409e-be91-4c47-b755-106b2cb2ba4f.png)
Found another place where weighting needs tweaking. Not tested fully but increasing the weight to 0.3 here solves this issue

Reply:
I'll take a look. I thought that just having a greater weighting on one string query would always weight it higher 🤷🏼‍♂️

Reply:
Yeah you would think that! It did fix the issue Jon found. I just made some quick changes and that fixed it. I can't quite remember exactly how the weighting works in Fuse.